### PR TITLE
Use Azure for deployment

### PR DESCRIPTION
**Pull request by syxolk**
I hope this actually works. Travis needs to push the current branch to Azure's master branch but I couldn't confirm yet that it's actually doing that.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,3 +4,6 @@ node_js:
 script:
 - npm run test
 - npm run lint
+deploy:
+  provider: azure_web_apps
+  on: develop
```
Comment (AhmetTanakol) [inline .travis.yml:9]: Don't we have username and password?

Comment (syxolk) [reply .travis.yml:9]: I setup a couple of environment variables in travis directly: https://travis-ci.org/Rostlab/JS18_ProjectB_Group3/settings

**Comment by syxolk** [comment]
I'm pretty confident it works this way. I did the same thing for group 1 and 4 and it worked. Your demo page is here: https://js2018-group3.azurewebsites.net/

**Review by AhmetTanakol — APPROVED** [review]

**Comment by AhmetTanakol** [comment]
Please update the branch

**Pull request merged**